packages/cli/src/config/trustedFolders.ts

```diff
@@ -181,11 +204,12 @@ function getWorkspaceTrustFromLocalConfig(
   }
 
   if (folders.errors.length > 0) {
-    for (const error of folders.errors) {
-      console.error(
-        `Error loading trusted folders config from ${error.path}: ${error.message}`,
-      );
-    }
+    const errorMessages = folders.errors.map(
+      (error) => `Error in ${error.path}: ${error.message}`,
+    );
+    throw new FatalConfigError(
+      `${errorMessages.join('\n')}\nPlease fix the configuration file(s) and try again.`,
```

Comment:
nit: isn't there only a single user level trusted folders file? If that is the case, remove the `(s)`.